ansible/roles/sshkey-host-access/tasks/main.yml

```diff
@@ -0,0 +1,73 @@
+---
+
+- name: ensure the existence of the user directory
+  file:
+    path: "{{ KEY_LOC_PATH }}"
+    owner: "{{ KEY_LOC_OWNER }}"
+    group: "{{ KEY_LOC_GROUP }}"
+    mode: 0700
+    state: directory
+  delegate_to: "{{ play_on }}"
```

Comment:
`play_on` is a host. The vars are defined in the playbook (`41_shell_access.yml`), which runs the role twice: once for gateone and once for guacamole. On the first play, `play_on` is `shell` which is the host for gateone (defined in hosts file). In the second play, it is `guac_server` which will be added to the hosts file in our variables repos.